test/Interop/Cxx/ergonomics/Inputs/implicit-computed-properties.h

```diff
@@ -27,16 +27,21 @@ struct NoNameVoidGetter {
 };
 
 struct LongNameAllLower {
-  int getfoo() const { return 42; }
+  mutable int value = 42;
+  int getfoo() const { return value; }
+  void setfoo(int v) const { value = v ;}
 };
 
 struct LongNameAllUpper {
-  int getFOO() const { return 42; }
-  int setFOO(int v) const {}
+    mutable int value = 42;
+  int getFOO() const { return value; }
+  void setFOO(int v) const {value = v ;}
 };
 
 struct LongNameMix {
```

Comment:
LongNameMix doesn't really make sense here. This shows the need for two different tests:
```
struct UpperCaseMix {
  int getFoo() const;
  void SetFoo(int);
}
```
And:
```
struct UpperCaseGetterSetter {
  ...
}
```